Source/WebInspectorUI/UserInterface/Test.html

```diff
@@ -149,6 +149,7 @@
     <script src="Models/CollectionTypes.js"></script>
     <script src="Models/Color.js"></script>
     <script src="Models/ConsoleCommandResultMessage.js"></script>
+    <script src="Models/ConsoleSnippet.js"></script>
```

Comment:
Should we have a test or two to test basic functionality, like making sure the default console snippet can run and returns the expected result?

Reply:
Hmm.

On the one hand, yes it would be good to have tests.

On the other hand, there's really nothing special about the functionality of this patch. It's built on top of existing stuff (e.g. `WI.RuntimeManager.prototype.evaluateInInspectedWindow`) and is really just about storing (BTW we don't support `WI.ObjectStore` inside tests to avoid any state across tests) and displaying the content.  I intentionally wanted this to be as similar to "copypaste the same text into the Console" as possible, just with a bit less "hassle" (i.e. a "Run" button to do that instead).  Also the default `WI.ConsoleSnippet` is currently just a big comment explaining what Console Snippets are with an expression for `"Hello World!"`, which I could totally see changing in the future if we find something better as the default.